context/app/static/js/components/Header/HeaderContent/style.js

```diff
@@ -1,14 +1,18 @@
 import styled from 'styled-components';
 import Button from '@material-ui/core/Button';
 import Divider from '@material-ui/core/Divider';
+import SvgIcon from '@material-ui/core/SvgIcon';
 
 const Spacer = styled.div`
   flex-grow: 1;
 `;
 
 const HeaderButton = styled(Button)`
-  margin-left: 10px;
   color: ${(props) => props.theme.palette.white.main};
+
+  button {
+    padding: 0px;
+  }
```

Comment:
I'm not sure this css element selector is having an effect. Since we are specifying `component={Link}` the element is actually an `a` and there is a `span` wrapping the text. I would suggest inspecting the elements in dev tools.